cli/internal/packagemanager/fixtures/pnpm-patches.json

```diff
@@ -2,9 +2,7 @@
   "name": "turborepo-prune-removes-patched",
   "version": "1.0.0",
   "packageManager": "pnpm@7.15.0",
-  "workspaces": [
-    "packages/*"
-  ],
+  "workspaces": ["packages/*"],
```

Comment:
does this fix the node linting failure I've been seeing in CI? (meant to get into it this morning)

Reply:
Yep!